### ISSUE TITLE
Something went wrong There's an issue and the page could not be loaded.

### ISSUE THREAD

**Issue by Vurtunetop** [mifi/SimpleInstaBot#215]
### I have 2 problems, one of which is that the bot doesn't work at all

**1 problem:
When I entered the login details, the bot starts, enters them, then a two-factor authorization window appears and I don’t have time to enter the code, the page just refreshes. Please fix this.

Problem 2 (the bot doesn't want to work at all):
When I enter the bot after entering all the data, it gives the error "Something went wrong
There's an issue and the page could not be loaded.". This error started yesterday, from 01/15/2023. How can I fix it? Please help**

![image](https://user-images.githubusercontent.com/98046245/212609088-d8299273-e24f-4a11-bab2-fbc8705f2ea2.png)

**Comment by c-chytoudis** [comment]
Hello, please try these steps below:

1. Go to C:\Users\<USERNAME>\AppData\Roaming\SimpleInstaBot, make a copy of **insta_username-followed, insta_username-unfollowed, insta_username-liked-photos and config** files and delete the rest.
 
2. Run the **SimpleInstaBot** latest version and you will see that it will generate the same files you deleted before.

3. Paste the files that you made a copy to replace the new generated empty files so the bot will continue from the last point.

4. Run the **SimpleInstaBot**. After the bot will automatically fill username and password fields you will see the 2FA digit code. If you wait a little, the bot will show a warning log and will redirect to login page so you have to fill username, password manually (copy/paste) and 2FA digit code too. With this warning the bot will give a 10 second timeout/pause so you have enough time to do the job manually and for the first time only.

**Comment by Vurtunetop** [comment]
> @c-chytoudisУ меня такая же проблема, ваши действия не помогают. @mifiКакие-нибудь советы?

+++

**Comment by oscarwilli** [comment]
i have the same problem since today, it was working until yesterday very good.

**Comment by CorujaBR** [comment]
> ### I have 2 problems, one of which is that the bot doesn't work at all
> **1 problem: When I entered the login details, the bot starts, enters them, then a two-factor authorization window appears and I don’t have time to enter the code, the page just refreshes. Please fix this.
> 
> Problem 2 (the bot doesn't want to work at all): When I enter the bot after entering all the data, it gives the error "Something went wrong There's an issue and the page could not be loaded.". This error started yesterday, from 01/15/2023. How can I fix it? Please help**
> 
> ![image](https://user-images.githubusercontent.com/98046245/212609088-d8299273-e24f-4a11-bab2-fbc8705f2ea2.png)

Your IP has flagged as spam by isntagram. Try another account with another IP and another UA.

**Comment by Vurtunetop** [comment]
> > ### У меня 2 проблемы, одна из которых в том, что бот вообще не работает
> > **1 проблема: Когда я ввел данные для входа, запускается бот, вводит их, потом появляется окно двухфакторной авторизации и я не успеваю ввести код, страница просто обновляется. Пожалуйста, исправьте это.
> > Проблема 2 (бот вообще не хочет работать): Когда захожу в бота после ввода всех данных, выдает ошибку "Что-то пошло не так. Возникла проблема и страница не может быть загружена.". Эта ошибка началась вчера, с 15.01.2023. Как я могу это исправить? Пожалуйста помоги**
> > ![изображение](https://user-images.githubusercontent.com/98046245/212609088-d8299273-e24f-4a11-bab2-fbc8705f2ea2.png)
> 
> Ваш IP-адрес был помечен как спам системой imstagram. Попробуйте другую учетную запись с другим IP и другим UA.
I have a dynamic ip address, it changes every time I reboot the router

**Comment by Vurtunetop** [comment]
@CorujaBR 
I have a dynamic ip address, it changes every time I reboot the router (I'm in Ukraine, I'm at war and we periodically turn off the electricity in order to give electricity to another part of the population). Usually earlier, when instagram noticed extreme activity, when switching to any profile (mine or someone else's, if I or the bot changed to someone else's) it was written that the page was deleted or blocked. This was written if the bot or instagram was opened ONLY FROM a PC, through the instagram application for mobile, everything worked fine. Changing the ip address then did not help either, I tried it. Now the error is thrown by the bot itself, as I understand it, or just the bot transmits this error with its interface, but changing the IP address DOES NOT HELP AT ALL

**Comment by mifi** [comment]
One thing to try is to log into instagram thru your browser (edge,chrome,safari) and see if that works. if not, it means that instagram probably has flagged the web app for the particular account(and ip?)

**Comment by JohanHuynh0130** [comment]
Hi there! I came across the same issue. It happened only whenever you're using your PC browsers (any browsers). I tried to change IP by using VPN, but it didn't work. The issue happened on any PCs.

**Comment by mifi** [comment]
I think instagram flags the account somehow

**Comment by flo28rin** [comment]
Did you ever get this fixed? It shows the error on the app and on my chrome browser. If I use incognito or another browser, it seems ok. Is this an actual flag from Instagram, would the account be at risk?

**Comment by JJRobertsSex30** [comment]
> Edit3: Second day, similar outcome. I lowered the limit, but the bot was still blocked after 6 hours of running. But this time I can still access instgram on my PC and all browsers.

I am having the same problem.

Did you ever find out what are the correct limits to set the bot at? Once that does not trigger this spam flagging?

**Comment by flo28rin** [comment]
> > Edit3: Second day, similar outcome. I lowered the limit, but the bot was still blocked after 6 hours of running. But this time I can still access instgram on my PC and all browsers.
> 
> I am having the same problem.
> 
> Did you ever find out what are the correct limits to set the bot at? Once that does not trigger this spam flagging?

No, I just started doing this manually. Have had less problems and it is faster, but I am mostly doing just follows.

**Comment by JJRobertsSex30** [comment]
> No, I just started doing this manually. Have had less problems and it is faster, but I am mostly doing just follows.

So you stopped using this bot completely?

**Comment by flo28rin** [comment]
> > No, I just started doing this manually. Have had less problems and it is faster, but I am mostly doing just follows.
> 
> So you stopped using this bot completely?

Yeah, I don't wanna risk my accounts

**Comment by JJRobertsSex30** [comment]
> Yeah, I don't wanna risk my accounts

Well my account is back to normal now so it was clearly only a temporary thing. I think just need to adjust the app settings. If anyone has ideal settings for the app, let me know.

**Comment by andystmartin** [comment]
@Vurtunetop i was using a vpn, when i turned off my instagram page worked. i think that might be the problem
